joomla/joomla-cms
# [HTTPS] Sample data

#### Summary of Changes

This PR changes all http:// URI in sample data that use sites exclusive in HTTPS to https://
#### Testing Instructions
1. Install Joomla staging with sample data from this PR (https://github.com/andrepereiradasilva/joomla-cms/archive/promote-https-sampledata.zip)
2. Check the URLs of sample data after install are HTTPS
#### Observations

After this PR, the only URI that are not in HTTPS in sample data are:
- http://forum.joomla.org/*
- http://extensions.joomla.org/*
- http://community.joomla.org/*
- http://resources.joomla.org/*
- http://feeds.joomla.org/*
- http://friendfeed.com/joomla (this seems to not exist anymore)

I didn't change this ones because they are still only HTTP or not exclusive HTTPS.
